tests/mcs/resource_manager/resource_manager_test.go

```diff
@@ -293,15 +293,7 @@ func (suite *resourceManagerClientTestSuite) TestResourceGroupController() {
 		re.Contains(resp, "Success!")
 	}
 
-	cfg := &controller.RequestUnitConfig{
```

Comment:
This configuration is for the convenience of construction consumption speed case. Don't delete it. Otherwise modify `tokenConsumptionPerSecond` in this test

Reply:
thx for ur advice.